modules/osdk-java-controller-memcached-deployment.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/java/osdk-java-tutorial.adoc
+
+:_content-type: CONCEPT
+[id="osdk-java-controller-memcached-deployment_{context}"]
+=  Defining `createMemcachedDeployment`
+
+The `createMemcachedDeployment` method described below will use the link:https://fabric8.io/[fabric8] `DeploymentBuilder` class:
```

Comment:
> ! below adjective, adverb, preposition
Do not use to refer to an earlier software version or fix level. Use "earlier". Do not use to indicate a relative location in a document, as in "the information below". Use "the information later in this section". Alternatively, you can use "following" to refer to a table, to art, or to a list. Example: "The following table shows ..."

> Write in active voice and the present tense as much as possible.